src/api/user-api.js

```diff
@@ -0,0 +1,19 @@
+import config from '~/config';
+import $ from 'jquery';
+
+export function login(username, password) {
+    const payload = {
+        email: username,
+        password,
+    };
+    return $.post(`${config.backend}/authentication/login`, payload);
+}
+
+export function checkAuth() {
+    const token = localStorage.getItem('auth_token');
+    return $.ajax({
+        type: 'GET',
+        url: `${config.backend}/authentication/status`,
+        beforeSend: (xhr) => { xhr.setRequestHeader('Authorization', token); },
```

Comment:
Since just the header is being set here, would it be better to use the `headers` option? http://api.jquery.com/jquery.ajax/#jQuery-ajax-settings